### PR TITLE
feat: comments PR with info for reviewers

### PR DESCRIPTION
**Pull request by benjaminParisel**
- Create a composite GitHub action
- Adding links for created/updated files
- Adding warning if some file is deleted or renamed on a PR

Covers #373 
Covers #463

### CODE REVIEW

**Comment by benjaminParisel** [comment]
This composite action could be tested on this PR https://github.com/bonitasoft/bonita-doc/pull/2212

:warning: Don't merge this PR on bonita-doc, when the composite action on bonita-documentation-site will be merged, this new behavior will be ready.

**Comment by benjaminParisel** [comment]
The comment added could contain a warning to remember to the user to add an alias if he deleted/renamed a source file.

Without warning about alias

image

With a warning about alias
image

Comment with deleted/renamed file

**Comment by tbouffard** [comment]
⚠️ when running the action as part of the workflow of the `bonita-documentation-site`, some errors occurs during the detection of the changed files.
The action is looking for files in the modules directory, and we don't have such directory here. We could skip this execution when running in this repository. But it can also occur when running in the documentation content repository (when we update the workflow calling the action for instance), so if possible, we should find a way to fix the problem in all cases (I didn't check if/how this is possible 😸).

```
changed-files
  Resolving repository path: /home/runner/work/bonita-documentation-site/bonita-documentation-site/.
  Retrieving changes between 884f629ce1d64e7cc97fc585b4c46455c0c08596 (master) → 504fe9c16edb00858941b0e55c7b7a83142accea (ci/composite_action_comment_links)
  grep: : No such file or directory
  /home/runner/work/_actions/tj-actions/changed-files/v34/get-changed-paths.sh: line 66: echo: write error: Broken pipe
  Error: Process completed with exit code 1.
```

File: .github/actions/comment-pr-with-links/action.yml
Comment (tbouffard) [inline .github/actions/comment-pr-with-links/action.yml:9]:
```suggestion
    description: 'The name of the component that is built. Used to create preview urls'
```

File: .github/actions/comment-pr-with-links/action.yml
Comment (tbouffard) [inline .github/actions/comment-pr-with-links/action.yml:11]: ⚠️ I would make the input required without default. The action is going to be used only internally, and we will always pass the component

File: .github/actions/comment-pr-with-links/action.yml
Comment (tbouffard) [inline .github/actions/comment-pr-with-links/action.yml:75]: Wording proposed by [Deepl](https://www.deepl.com/translator)
```suggestion
            warningAliasMessage='\n \n ### :warning: At least one file is deleted in this Pull Request, be sure to add [aliases](https://github.com/bonitasoft/bonita-documentation-site/blob/master/docs/content/CONTRIBUTING.adoc#use-alias-to-create-redirects)'
```

File: .github/actions/comment-pr-with-links/action.yml
Comment (tbouffard) [inline .github/actions/comment-pr-with-links/action.yml:74]:
```suggestion

```

File: .github/actions/comment-pr-with-links/action.yml
Comment (tbouffard) [reply .github/actions/comment-pr-with-links/action.yml:11]: Let keep the default value.

**Review by tbouffard — APPROVED** [review]
LGTM, great job

**Pull request merged**
